src/core/annotation.js

```diff
@@ -1081,6 +1094,92 @@ class WidgetAnnotation extends Annotation {
 
     return `${shift} ${vPadding} Td (${text}) Tj`;
   }
+
+  handleMultiline(text, font, fontSize, width, alignment, hPadding, vPadding) {
+    const lines = text.replace("\r\n", "\n").split("\n");
+    let buf = "";
+    const totalWidth = alignment === 1 ? width : width - 2 * hPadding;
```

Comment:
This could use a comment. What is `alignment === 1`; center, left of right? Why is that special cased?